docs/README.md

```diff
@@ -19,42 +19,42 @@ The props below support [`responsive-values`](https://github.com/vtex-apps/respo
 
 ### `flex-layout.row`
 
-| Prop name                  | Type                  | Description                                                                                                   | Default value |
```

Comment:
Reorder the props here and on `.column` too please :D

Reply:
Sure!